### ISSUE TITLE
Kivy Anaconda not working (version 1.9.0rev3) on OSX

### ISSUE THREAD

**Issue by tabchas** [kivy/kivy#3768]
I tried to get Anaconda working with Kivy 1.9.0 following the guidelines here: https://github.com/kivy/kivy/wiki/Connecting-Kivy-with-Anaconda-(OSX)

When I try to run kivyconda with a sample kivy program, I get: 

    Tabishs-MBP:Desktop tabchas$ kivyconda test.py 
    discarding /Users/tabchas/anaconda2/bin from PATH
    prepending /Users/tabchas/anaconda2/envs/kivy/bin to PATH
    [INFO   ] [Logger      ] Record log in /Applications/Kivy.app/Contents/Resources/.kivy/logs/kivy_15-   11-14_1.txt
    [INFO   ] [Kivy        ] v1.9.0
    [INFO   ] [Python      ] v2.7.10 |Continuum Analytics, Inc.| (default, Oct 19 2015, 18:31:17) 
    [GCC 4.2.1 (Apple Inc. build 5577)]
    [INFO   ] [Factory     ] 173 symbols loaded
    [INFO   ] [Image       ] Providers: img_tex, img_imageio, img_dds, img_gif (img_sdl2, img_pil,     img_ffpyplayer ignored)
    [CRITICAL] [Window      ] Unable to find any valuable Window provider at all!
    sdl2 - ImportError:     dlopen(/Applications/Kivy.app/Contents/Resources/kivy/kivy/core/window/_window_sdl2.so, 2): Library     not loaded: @executable_path/../Frameworks/SDL2.framework/Versions/A/SDL2
    Referenced from: /Applications/Kivy.app/Contents/Resources/kivy/kivy/core/window/_window_sdl2.so
    Reason: image not found
    File "/Applications/Kivy.app/Contents/Resources/kivy/kivy/core/__init__.py", line 57, in core_select_lib
    fromlist=[modulename], level=0)
    File "/Applications/Kivy.app/Contents/Resources/kivy/kivy/core/window/window_sdl2.py", line 26, in <module>
    from kivy.core.window._window_sdl2 import _WindowSDL2Storage

    [CRITICAL] [App         ] Unable to get a Window, abort.

**Comment by sirpercival** [comment]
I've actually found that once I've set kivy up in this way, I can just run kivy with anaconda's python. What happens when you try that?

**Comment by tabchas** [comment]
That does not work for me since when I try to import kivy using anaconda's python, it gives an error. So you don't need to use the kivyconda script?

**Comment by tabchas** [comment]
Just did a clean install of Anaconda and tried again. Still same issue. I also did some searching and the typical reason for this error is the fact that pygame is not installed. But pygame is built into the Kivy app correct?

**Comment by tabchas** [comment]
@sirpercival Hey, do you think you could try to reproduce on your machine? Thanks!

**Comment by trentoliphant** [comment]
I have run into the same problem.  I even did a conda install of pygame into the environment to see if that would help - it didn't.

**Comment by dessant** [comment]
@tabchas @trentoliphant pygame is not used since 1.9.0, it has been deprecated in favor of sdl2.

**Comment by tabchas** [comment]
@trentoliphant I ended up just following the instructions here: http://kivy.org/docs/installation/installation-osx.html and installing directly into anaconda. @dessant It still required me to install pygame however.

**Comment by dessant** [comment]
Then something might be wrong, pygame should not be needed and sdl2 is shipped if you use the portable package.

**Comment by trentoliphant** [comment]
@tabchas Are you referring to the manual instructions at the bottom of the page?  I'm not quite sure I understand what you mean by installing directly into anaconda.  

I had installed Kivy according to the instructions at the top of the page.  I am able to run Kivy just fine using the script included in the .app package.  When I do the steps outlined in the Anaconda post mentioned above I get the error if I run it with the kivyconda script.

**Comment by tabchas** [comment]
@trentoliphant Yes I am referring to the manual instructions at the bottom of the page. The steps outlined in the Anaconda post is showing a way to bootstrap the Kivy environment variables into an Anaconda environment. If this was to work, you would not need to install Kivy and its dependencies manually. You would simply grab the .app package, and bootstrap Kivy's environemnt variables into Anaconda.

Since this doesn't work, I followed the instructions to install Kivy manually into my Anaconda environment. This does require installing dependencies manually and pygame, as described in the instructions I linked before. This works perfectly for me.
